compiler/noirc_evaluator/src/ssa/opt/flatten_cfg.rs

```diff
@@ -662,6 +662,22 @@ impl<'f> Context<'f> {
                     self.remember_store(address, value);
                     Instruction::Store { address, value }
                 }
+                Instruction::RangeCheck { value, max_bit_size, assert_message } => {
```

Comment:
Is there a performance difference if we simply multiplied by `current_side_effects_enabled_var` in acir gen when converting an `Instruction::RangeCheck` rather than inserting a mul instruction here which requires us to also add a cast?

Reply:
the cast should not impact acir opcodes since we cast from a value of 1 bit

Reply:
Yep, the cast doesn't affect the outputted ACIR as we're expanding the bit size.